aten/src/ATen/native/BlasKernel.cpp

```diff
@@ -180,9 +181,10 @@ void gemv(char trans, int64_t m, int64_t n, scalar_t alpha, scalar_t *a, int64_t
     return;
   }
 
+  using opmath_t = at::opmath_type<scalar_t>;
   if ((trans == 'T') || (trans == 't')) {
     for (const auto i : c10::irange(n)) {
-      scalar_t sum = 0;
+      opmath_t sum = 0;
```

Comment:
Just noticed `beta` and `alpha` are only `scalar_t` here. In future it would be nice for that to be `opmath_t` but it's okay as-is for this PR